cfme/common/provider.py

```diff
@@ -710,8 +710,11 @@ def validate_stats(self, ui=False):
             return
         else:
             # Set off a Refresh Relationships
-            method = 'ui' if ui else None
-            self.refresh_provider_relationships(method=method)
+            if self.category == "config_manager" and ui:
```

Comment:
Why go the route of making checks and changing function calls based on the category, instead of updating ConfigManager base class to be consistent with the BaseProvider methods?

Reply:
Talked with Parthvi about this, due to the level of effort in updating BaseProvider/ConfigManager base class, I'm going to prioritize the test coverage over rejecting these customization within BaseProvider.